app/src/main/java/ar/com/wolox/android/example/ui/home/HomeFragment.kt

```diff
@@ -0,0 +1,33 @@
+package ar.com.wolox.android.example.ui.home
+
+import android.content.Intent
+import android.os.Build
+import androidx.annotation.RequiresApi
+import ar.com.wolox.android.R
+import ar.com.wolox.android.example.ui.login.LoginActivity
+import ar.com.wolox.android.example.utils.onClickListener
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import kotlinx.android.synthetic.main.fragment_home.*
+
+class HomeFragment : WolmoFragment<HomePresenter>(), IHomeView {
+
+    override fun layout(): Int = R.layout.fragment_home
+
+    override fun init() {
+    }
+
+    override fun setListeners() {
+        vHomeLogOutButton.onClickListener {
```

Comment:
I think there is no logout button in the design . You add this just for testing ?

Reply:
Sorry, I add the logout button for testing and I forgot to take it out.